datalad/support/network.py

```diff
@@ -611,13 +612,21 @@ class URL(RI):
 
     _FIELDS = RI._FIELDS + (
         'scheme',
+        'netloc',
         'username',
         'password',
         'hostname', 'port',
         'query',
         'fragment',
     )
 
+    # Only interpreted on Windows. If set to `True`, UNC-names encoded in
+    # file-URLs, e.g. "file://server/share/path", would be considered local
+    # and mapped onto "\\server\share\path". If set to `False`, UNC-names
+    # encoded in file-URLs would not be considered local and cannot be resolved
+    # to a local path.
+    support_unc = False
```

Comment:
The value is to be set by developers or some generic config processing. It should not yet be in a configuration because datalad does not support it yet, and  I am not sure about `git-annex` and Windows shares. So there is currently no point in activating UNC support. Also, if we should support it in the future, there is probably no need to deactivate it from a configuration.

With this in mind, the line documents that UNC could be supported by the class, if the users of the class can handle it. (And I think it is good to give a `False` a name that indicates its purpose. In addition, it allows us to be able to switch UNC support on and off in a single place if we decide to support it in general (no config-option)).

Reply:
Still not sure why we would have it here then, but ok, let it be.